src/libultra/io/pirawdma.c

```diff
@@ -1,23 +1,23 @@
 #include "global.h"
 
 s32 __osPiRawStartDma(s32 direction, u32 devAddr, void* dramAddr, size_t size) {
-    register int stat;
+    register int status;
```

Comment:
Personally I think the code looks cleaner like this but I don't think we have a set standard:
```suggestion
    register int status = HW_REG(PI_STATUS_REG, u32);
```